python/insights_ansible_playbook_lib/__init__.py

```diff
@@ -204,3 +204,13 @@ def get_revocation_digests(playbook: str, gpg_key: bytes) -> set[bytes]:
     revoked: list[dict] = play.get("revoked_playbooks", [])
     digests = set(bytes(bytearray.fromhex(item["hash"])) for item in revoked)
     return digests
+
+
+def represent_none(self: yaml.Dumper, data: None) -> yaml.ScalarNode:
+    """Represent null as an empty string.
+
+    :param self: The YAML object.
+    :param data: The data to be represented.
+    :returns: An empty string.
+    """
+    return self.represent_scalar("tag:yaml.org,2002:null", "")
```

Comment:
The usage of `represent_none()` is done manually both in the signer and in each test, and that makes it easy to miss/forget.

My suggestion here would be to create somewhere (see the other comment of @m-horky) something similar to what done for the `Loader`, i.e. create a subclass of `yaml.Dumper` that is properly set up.